DOCS-1447: Add info to components and services landing pages (#2641)
viamrobotics/docs: docs/services.md

Services are built-in software packages that make it easier to add complex capabilities such as motion planning or object detection to your machine.

> {{< imgproc src="/viam/machine-components.png" alt="Machine components" resize="1000x" style="max-width:650px" >}}

Reviewer: I think centering these might look better. I'm a bit concerned that it's not clear here what services are. Maybe the previous text should reference the services in this image

Author: Ok! I've requested a "SERVICES" label inside the services box for this and the other diagram. I'm also adding additional copy above here--it might be overkill though; I fear it makes this page too verbose. lmk what you think.

<br>

> Even though many services run locally within `viam-server`, you can think of them as discrete building blocks; you can run your machine using none, some, or all of them, depending on your use case.

Reviewer: "Even though" is generally used to introduce a seeming contradiction - I don't think there's a contradiction here.

```suggestion
You can think of services as discrete building blocks; you can run your machine using none, some, or all of them, depending on your use case.
```

Author: Yes--I do see it as a contradiction because it's software that is already there, installed on your robot, as part of RDK; you're not actually "adding" anything--you're technically just choosing to use more of what is already there on your robot.

Reviewer: Ok - if I stumble over that I am not sure it's clear enough. Can you try to make it clearer? Here's my attempt but feel free to change:
```suggestion
Many builtin services run locally within `viam-server`, but you can think of them as discrete building blocks that you can mix and match however you want; you can run your machine using none, some, or all of them, depending on your use case.
```
I think this still keeps the contradiction but for people who don't see it as a contradiction (I can see something as building blocks even if its within something else...) it might be better? Open to alternatives

Author: ```suggestion
Many built-in services run locally within `viam-server`, but you can think of them as discrete building blocks that you can mix and match however you want; you can run your machine using none, some, or all of them, depending on your use case.
```

> Configure just the services you want to use.

Reviewer: The previous sentence covers that sufficiently I think
```suggestion
```

Author: I put that in here in an attempt to transition to the next paragraph about configuration, and to explain *how* someone would use some, none or all services seeing as they're already on their machine.

Author: Okay--I think we can make this more concise and the next paragraph covers it well enough beyond this:
```suggestion
To use a given service, add it to your machine's configuration.
```
